src/components/TodoList/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './TodoList';
```

Comment:
Everything seems to be OK with the code provided for the src/components/TodoList/index.tsx file. Good job! 😃